docs/pages/fund-agents-apps/calculate-fees.mdx

```diff
@@ -43,15 +43,15 @@ Use this guide to understand XMTP fees and how to calculate estimated XMTP fees
 
 ## Understand XMTP fees
 
-To support a decentralized and sustainable network, XMTP operates on a usage-based fee model. The fees paid by apps and agents (payers) directly compensate the independent node operators who run the infrastructure, ensuring the network remains resilient, secure, and censorship-resistant.
+To support a decentralized and sustainable network, XMTP operates on a usage-based fee model. **All XMTP network fees are paid in USDC** (USD Coin, a stablecoin pegged to the US dollar). The fees paid by apps and agents (payers) directly compensate the independent node operators who run the infrastructure, ensuring the network remains resilient, secure, and censorship-resistant.
```

Comment:
Added USDC usage here and on line 73. 

However, I left references to "$X" on lines 50, 52, 54, 130, 132, 133, 134, and 177. Should I change these to use "X USDC" instead? Wasn't sure how strict we want to be vs readability.